app/assets/src/components/views/samples/SamplesView.jsx

```diff
@@ -192,22 +193,63 @@ class SamplesView extends React.Component {
         ? mapSidebarSelectedSampleIds
         : selectedSampleIds;
 
-    const log = () =>
-      logAnalyticsEvent("SamplesView_heatmap-icon_clicked", {
-        selectedSampleIds: targetSampleIds.length,
+    const heatmapOptions = [
+      { text: "Taxon Heatmap", value: "/visualizations/heatmap" },
+    ];
+
+    if (this.props.admin) {
+      heatmapOptions.push({
+        text: "AMR Heatmap",
+        value: "/amr_heatmap",
       });
-    return targetSampleIds.size < 2 ? (
-      <HeatmapIcon className={cx(cs.icon, cs.disabled, cs.heatmap)} />
-    ) : (
-      <a
-        onClick={log}
-        href={`/visualizations/heatmap?sampleIds=${Array.from(
-          targetSampleIds
-        )}`}
-      >
-        <HeatmapIcon className={cx(cs.icon, cs.heatmap)} />
-      </a>
-    );
+
+      return targetSampleIds.size < 2 ? (
+        <HeatmapIcon className={cx(cs.icon, cs.disabled, cs.heatmap)} />
+      ) : (
+        <BareDropdown
+          hideArrow
+          className={cx(cs.icon, cs.heatmapDropdown)}
+          closeOnClick={true}
```

Comment:
Nit: closeOnClick is already `true` by default (see BareDropdown's `defaultProps`) so no need for this. Also, when setting a boolean prop to true, you can just do `closeOnClick` instead of `closeOnClick={true}`.